Makefile

```diff
@@ -170,6 +172,17 @@ run-notebook-readflow-tests-katalog:
 	$(MAKE) setup-cluster
 	$(MAKE) -C manager run-notebook-readflow-tests
 
+.PHONY: run-network-policy-readflow-tests-katalog
+run-network-policy-readflow-tests-katalog: export HELM_SETTINGS=--set "coordinator.catalog=katalog" --set "global.isNPEnabled=true"
```

Comment:
isNPEnabled is not defined in the master branch

Reply:
We will wait to merge PR #2051 that defines the `isNPEnabled` flag.